docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -197,7 +197,26 @@ implementation is used:
   - Set to `false` to disable in-memory caching of metadata files on the 
     coordinator. This cache is not used when `fs.cache.enabled` is set to true.
   - `true`
-
+* - `iceberg.expire_snapshots.min-retention`
+  -  Minimal retention period for `expire_snapshot` procedure. Equivalent
+     session property is `expire_snapshots_min_retention`.
+  -  `7 DAYS` 
+* - `iceberg.remove_orphan_files.min-retention`
+  -  Minimal retention period for `remove_orphan_files` procedure Equivalent
+     session property is `remove_orphan_files_min_retention`.
+  -  `7 DAYS`
+* - `iceberg.idle-writer-min-file-size`
+  -  Minimum data written by a single partition writer before it can
```

Comment:
Thats an implementation details .. what is the effect to the user and why would they configure this. Also link to https://trino.io/docs/current/admin/properties.html#data-size

Reply:
https://github.com/trinodb/trino/pull/19649

@raunaqmorarka  can you pls help me summarize to explain what this config is doing?

Reply:
It potentially helps to reduce memory usage for writes to partitioned tables https://github.com/trinodb/trino/pull/19649
It's very low level detail, i'm not sure there is a nicer way to explain it.